core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -493,13 +493,13 @@ class ContainerPool(
         val network = config.invokerContainerNetwork
         val policy = config.invokerContainerPolicy
         val env = getContainerEnvironment()
-        val pull = !imageName.contains("whisk/")
+        val pull = !imageName.contains(config.dockerImagePrefix + "/")
```

Comment:
since `action: WhiskAction` is in scope, should this be replaced by `action.exec is blackbox` instead?